sdk/spring/spring-test-template.yml

```diff
@@ -10,6 +10,8 @@ parameters:
     - spring/azure-spring-cloud-test-eventhubs
     - spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder
     - spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-kafka
+    - spring/azure-spring-boot-test-aad
+    - spring/azure-spring-boot-test-aad-b2c
```

Comment:
I think there are no testing resources defined in aad and aad b2c project?

Reply:
oh, I understand wrong, so we can remove test-aad too.